Source/WebKit/Scripts/webkit/messages.py

```diff
@@ -272,6 +273,19 @@ def message_to_struct_declaration(receiver, message):
         result.append('\n        : m_arguments(%s)\n' % ', '.join(arguments_constructor_parameters))
         result.append('    {\n')
         result.append('    }\n\n')
+
+    if message.coalescing_key_indices is not None:
+        result.append('    // Not valid to call this after arguments() is called.\n')
+        result.append('    void encodeCoalescingKey(IPC::Encoder& encoder) const\n')
+        result.append('    {\n')
+        if len(message.coalescing_key_indices) > 0:
+            get_arguments_string = ' << '.join((f'std::get<{i}>(m_arguments)' for i in message.coalescing_key_indices))
+            result.append(f'        encoder << {get_arguments_string};\n')
+        else:
```

Comment:
nit: Mark the argument with `[[maybe_unused]]`

Reply:
I'll just change it so that we explicitly emit `encodeCoalescingKey(IPC::Encoder& encoder)` or `encodeCoalescingKey(IPC::Encoder&)` depending on whether the param is used and get rid of the `UNUSED_PARAM` thing.

Reply:
Nice!